extensions/barrage/src/main/java/io/deephaven/extensions/barrage/table/BarrageTable.java

```diff
@@ -217,16 +218,18 @@ public BitSet getServerColumns() {
      * @param onSealFailure pass a callback that gets invoked if the table fails to finish applying updates
      */
     public synchronized void sealTable(final Runnable onSealRunnable, final Runnable onSealFailure) {
```

Comment:
Do we still need sealTable? If we're not refreshing, we don't need to use the update graph at all.